certified-connectors/Leadinfo/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "[DUMMY_VALUE]",
+          "scopes": [
+            "webhook"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://portal.leadinfo.com/oauth2"
+            },
+            "tokenUrl": {
+              "value": "https://api.leadinfo.com/oauth2/token"
+            },
+            "refreshUrl": {
+              "value": "https://api.leadinfo.com/oauth2/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FA2E48",
+    "capabilities": [],
+    "publisher": "Leadinfo"
```

Comment:
@GerbenZ01 Please add the "stackOwner" property. please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).
```
"stackOwner": "Name of the owner of the service or product."
```